airbyte-integrations/connectors/build.gradle

```diff
@@ -98,4 +98,26 @@ allprojects {
     // Convenience tasks for local airbyte-ci execution.
     airbyteCIConnectorsTask('airbyteCIConnectorBuild', 'build')
     airbyteCIConnectorsTask('airbyteCIConnectorTest', 'test')
+
+    // Task to build connector image using the CDK CLI
+    tasks.register('buildImage') {
+        description = 'Build connector Docker image using the CDK CLI'
+        group = 'build'
+
+        doLast {
+            logger.lifecycle("Building connector image using airbyte-cdk...")
+            logger.lifecycle("If this fails, make sure `uv` is installed (`brew install uv`).")
+
+            // Run the command directly
+            def result = exec {
+                workingDir = projectDir
+                executable 'uvx'
+                args 'airbyte-cdk[dev]@latest', 'image', 'build'
+                ignoreExitValue = false
+            }
+        }
+
+        // Always re-run this task
+        outputs.upToDateWhen { false }
```

Comment:
checking: we can't use the [same plugin](https://github.com/airbytehq/airbyte-platform-internal/blob/6a0ef747e2a6ade4e71a3981861f978c773e449a/oss/airbyte-connector-builder-server/build.gradle.kts#L1) that platform uses because we don't have actual Dockerfiles sitting around?

(I think that would get us smarter caching behavior)